core/src/main/resources/hudson/model/ComputerSet/sidepanel.jelly

```diff
@@ -30,7 +30,9 @@ THE SOFTWARE.
   <l:header />
   <l:side-panel>
     <l:tasks>
-      <l:task href="../configureClouds" icon="symbol-cloud" permission="${app.SYSTEM_READ}"
+      <l:task href="${h.getNearestAncestorUrl(request,it)}/" icon="symbol-computer"
+              title="${%Nodes}"/>
+      <l:task href="../cloud/" icon="symbol-cloud" permission="${app.SYSTEM_READ}"
```

Comment:
This looks slightly awkward when lacking permissions to access clouds or node monitoring, but is a reasonable enough tradeoff IMO.